ArduPlane/sensors.cpp

```diff
@@ -117,6 +123,12 @@ void Plane::update_sensor_status_flags(void)
         control_sensors_present |= MAV_SYS_STATUS_SENSOR_OPTICAL_FLOW;
     }
 #endif
+
+#if PRECISION_LANDING == ENABLED
+    if (precland.enabled()) {
+        control_sensors_present |= MAV_SYS_STATUS_SENSOR_VISION_POSITION;
```

Comment:
This is definitely not the correct sensor to be using.  It's no more incorrect that Copter, however.

Reply:
Which sensor should be use?

Reply:
Very much an "out of scope" thing for your PR here, so don't feel indebted to do anything here - but we probably need a new sensor for it!

This is the existing list of sensors: https://github.com/ArduPilot/mavlink/blob/master/message_definitions/v1.0/common.xml#L456

None of them seem appropriate for all of the different precland backends we support.